extractors/main.py

```diff
@@ -4,18 +4,27 @@
 
 from datetime import datetime
 
+import sys, getopt
 
-def main():
-    cnac = ExtractCNAC()
-    cnac.download()
 
-    rciv = ExtractRCIV(
-        initial_date=datetime(2021, 1, 1), final_dalte=datetime(2021, 2, 28)
-    )
-    rciv.download()
+def main(source):
+    if source == "CNAC":
+        cnac = ExtractCNAC()
+        cnac.download()
+    elif source == "RCIV":
+        rciv = ExtractRCIV(
+            initial_date=datetime(2021, 1, 1), final_dalte=datetime(2021, 2, 28)
+        )
+        rciv.download()
+    elif source == "SRAG":
+        srag = ExtractSRAG()
+        srag.download()
```

Comment:
I'm down with @brbraziel suggestion. This way we can make it generic and avoid an overload of conditionals in here:

```suggestion
import json
import os
import importlib.util

def main(source_path, source_args):
    try:
       source_name = os.path.splitext(os.path.basename(source_path))[0]
       spec = importlib.util.spec_from_file_location(source_name, source_path)
       module = importlib.util.module_from_spec(spec)
       module_inst = module(*source_args)
       module_inst.download()
    except:
       print('dynamic import not successful')
```

Reply:
Just the idea here, I didn't test to see if this dynamic import works, especially with **multiple args**.